### ISSUE TITLE
Is this a GPT-3 bot experiment?

### ISSUE THREAD

**Issue by shawwwwwwww** [botupdate/botupdate#106]
Just wondering... are YOU a bot? Because this is some serious Deep Dream GPT-3 dark corner of the internet research experiment business. It seems like a likely context for a GPT-3 experiment to be warning us about the misuse of neural nets to brainwash us, just saying.

**Comment by o2-b** [comment]
Please unsubscribe.

**Comment by Venryx** [comment]
@shawwwwwwww That's a good point. Would be very funny if that turned out to be the case.

That said, take a look at the other repos by the creator (the "botupdate" user). [One of his repos](https://github.com/botupdate/video) has hours upon hours of videos that he claims to be of himself (I watched some of them out of curiosity), talking to invisible entities he hears in his head.

So my guess is that the author, at least, is schizophrenic, and this repo is his attempt at sharing the worldview he's developed from those experiences, and the psychological methods he uses to try to relieve the discomfort.

**Comment by wnfo21** [comment]
No…sorry to disappoint have no clue what any of this is but It’s on my computer being used for something and I’d like to find out for what reason ha and I thought I just knew a thing or two about computers

**Comment by Venryx** [comment]
@wnfo21 When did you first find out about this "botupdate" repo?

**Comment by wnfo21** [comment]
Via your email- I don’t know what that is - I just put out there is there away to remove everything someone has put on with out removing the history clearly there has been calls and streaming of some sort going on

**Comment by Venryx** [comment]
@wnfo21 But *when* did you find out about this "botupdate" repository? Like, a week ago? A month ago?

**Comment by wnfo21** [comment]
Um I found all this mess on my computer signed up in hopes of some one helping figure out how to undo whatever he has put on here to hide what’s hidden… maybe 3/4 days ago not sure how I got in this bot update whatever-

**Comment by wnfo21** [comment]
I found some ***@***.*** site in hopes of uncovering anything and it just showed pretty much I think how to stream? Not 100%   And I think he as been using Google slide to do whatever it is but I can’t get it to pull anything up I can just see like pieces… I feel like he came here to find the codes to make it unrecognizable/hidden then put on my computer

**Comment by wnfo21** [comment]
Oh and of course when I questioned I’m the crazy one - but I can read and we are the only 2 with access to my computer and if I didn’t put it on there ahhh there only other person than can  - well anyway I’d just like to know what so important that needs to be hidden in this manner to go to this extreme- I didn’t even know anything about a developer tools or codes or any before this

**Comment by Venryx** [comment]
> we are the only 2 with access to my computer and if I didn’t put it on there ahhh there only other person than can

Who is the other person that has access to your computer? (if you don't mind saying)

(I have lots of other questions, and will do my best to help both of us understand this situation better, but I'll start with that question above.)

**Comment by wnfo21** [comment]
husband….

**Comment by Venryx** [comment]
> well anyway I’d just like to know what so important that needs to be hidden in this manner to go to this extreme

What made you believe that your husband was hiding something? Did you see something specific on your computer?

**Comment by wnfo21** [comment]
The word hidden a lot…. Some how I got the Mona Lisa  name to become visible clearly says call with the entire a-z behind it says to go to www.w3.org/1999/xhtm/ alot

**Comment by wnfo21** [comment]
Oh and he said it up for like remote access….. ?! It’s a lap top why would you need remote access you can take it anywhere

**Comment by wnfo21** [comment]
And I’d get on it now but I took it to work with me - he just keeps saying you can’t read any of it why are you trying to I removed it from the house- oh and Boolean alot says access codes alot but you can’t see them

**Comment by wnfo21** [comment]
Clearly this has nothing to do with whatever that botupdate is…. Not sure how it even got attached to anything maybe I hit the wrong button who knows - but any help on unhidding whatever coding this is would be greatly appreciated or how to see all the history

**Comment by Venryx** [comment]
@wnfo21 Are you responding to this thread from the main Github website? (ie. with your browser showing "github.com" at the top address-bar)

Or are you responding through your email service? (ie. with your browser showing gmail.com, yahoo.com, or the like in the address bar)

**Comment by wnfo21** [comment]
This is coming directly to my email

**Comment by LunarDevelopment** [comment]
@wnfo21 was your husband a marine and lives in the San Diego Area by any chance?

Your email chain that we're chatting in is just a fast way of accessing our online chat on the following URL: https://github.com/botupdate/botupdate/issues/106   you'll see that your emails match the posts here. 

This botupdate is a (harmless to your computer) strange set of files being stored on this website https://github.com/botupdate/botupdate - which is a bit like google drive, microsoft one drive, drop box etc (just online storage for techys). 

To address a few other things: 
- Remote access on a laptop is perfectly normal for file access. 
- https://www.w3.org/1999/xhtm/ isn't an ominous URL, it's referenced in thousands of files on your computer because it's a document protocol - liken it to checking a glossary in a book you're reading -  it's how your computer understands the contents of some of it's technical files. 
- All computers are full of "hidden" files and if a layman (or even techy) was to unhide them all it'd be an annoyance. 
- I reckon the most likely reason a man would be cagey about someone going around their laptop is that there is / has been porn on there.
